samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -25,12 +25,20 @@ NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
 static struct bt_conn *default_conn;
 static struct bt_audio_stream streams[MAX_PAC];
 
+
+static uint8_t unicast_server_addata[] = {
+	0x4E, 0x18, /* ASCS UUID */
+	0x01, /* Target Announcement */
+	0xFF, 0x00, 0xFF, 0x00, /* Available Audio Context */
+	0x00, /* Metadata length */
+};
+
 /* TODO: Expand with BAP data */
 static const struct bt_data ad[] = {
-	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
```

Comment:
We should still include these flags in the advertising data

Reply:
> We should still include these flags in the advertising data

OK, will do.